Cartfile.private

```diff
@@ -1 +1,2 @@
-github "https://github.com/futuretap/InAppSettingsKit.git" ~> 3.0
+github "braintree/braintree_ios" ~> 5.3
```

Comment:
random: is this `Cartfile.private` file still needed?

Reply:
We're still using Carthage in our demo app. We use Carthage to fetch InAppSettingsKit & Braintree dependencies in our Demo app. (The Demo app pulls in the local version of BraintreeDropIn).

We might update the Demo to use SPM in a future PR.

Reply:
Ahh TIL